modules/common/react/lib/styles/errorRing.ts

```diff
@@ -1,17 +1,35 @@
+import {CanvasTheme} from '@workday/canvas-kit-labs-react-core';
 import {ErrorType} from '../types';
 import {CSSObject} from '@emotion/core';
 import {colors, inputColors} from '@workday/canvas-kit-react-core';
+import chroma from 'chroma-js';
 
-export default function errorRing(error?: ErrorType): CSSObject {
+export default function errorRing(error?: ErrorType, theme?: CanvasTheme): CSSObject {
   let errorBorderColor;
   let errorBoxShadow;
 
+  const isAccessibleThemeColor = (themeColor: CanvasTheme): boolean => {
+    return chroma.contrast(themeColor, colors.frenchVanilla100) < 3;
```

Comment:
if it's above 3, it's accessible? i was trying to understand https://www.w3.org/TR/WCAG20-TECHS/G18.html
so should the number be 4? @anicholls

Reply:
Not sure you need to factor this into a separate function just for one line, but with the way you've written it, it would be `>= 3`. Anything 3 or above passes. You can say do nothing if it's 3 or above, or make a special case if it's less than 3.